### PR TITLE
[preview] Configure with-dedicated-emulation

### PR DESCRIPTION
**Pull request by geropl**
## Description

This PR:
 - disables SaaS related secrets (linkedin, stripe) if `with-dedicated-emulation` is set
 - disable the Onboarding flow for the `admin-user` ([commit](https://github.com/gitpod-io/gitpod/pull/17337/commits/c7b2a62cd17b7492181d67c746cecadc55289b64)): Not very fond of the solution, and definitely open for alternative suggestions!

## Related Issue(s)

Fixes WEB-213

## How to test
 - `previewctl admin credentials create`
 - visit `https://gpl-dedica693133f791.preview.gitpod-dev.com/api/login/ots/admin/your-token`
 - notice the "new org" screen :heavy_check_mark: 

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - gpl-dedica693133f791</li>
	<li><b>🔗 URL</b> - <a href="https://gpl-dedica693133f791.preview.gitpod-dev.com/workspaces" target="_blank">gpl-dedica693133f791.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [x] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

File: components/gitpod-protocol/src/protocol.ts [hunk 1]
```diff
@@ -123,6 +123,11 @@ export namespace User {
         if (!!user.organizationId) {
             return false;
         }
+        // If a user has already been onboarded
+        // Also, used to rule out "admin-user"
+        if (!!user.additionalData?.profile?.onboardedTimestamp) {
```
Comment (AlexTugarev) [inline components/gitpod-protocol/src/protocol.ts:128]: that's a weak signal, as it's free to change for everyone, no?

Comment (svenefftinge) [reply components/gitpod-protocol/src/protocol.ts:128]: I believe that is a totally fine approach.

File: dev/preview/workflow/preview/deploy-gitpod.sh [hunk 2]
```diff
@@ -369,24 +368,30 @@ fi
...
 # configureStripeAPIKeys
 #
-kubectl --kubeconfig "${DEV_KUBE_PATH}" --context "${DEV_KUBE_CONTEXT}" -n werft get secret stripe-api-keys -o yaml > stripe-api-keys.secret.yaml
-yq w -i stripe-api-keys.secret.yaml metadata.namespace "default"
-yq d -i stripe-api-keys.secret.yaml metadata.creationTimestamp
-yq d -i stripe-api-keys.secret.yaml metadata.uid
-yq d -i stripe-api-keys.secret.yaml metadata.resourceVersion
-diff-apply "${PREVIEW_K3S_KUBE_CONTEXT}" stripe-api-keys.secret.yaml
-rm -f stripe-api-keys.secret.yaml
+if [[ "${GITPOD_WITH_DEDICATED_EMU}" != "true" ]]
```
Comment (svenefftinge) [inline dev/preview/workflow/preview/deploy-gitpod.sh:371]: should this be == "true"?

**Review by svenefftinge — APPROVED** [review]
LGTM

**Pull request merged**